src/components/graphic.jsx

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
-import PropTypes from 'prop-types';
+
 import { Grid, GridContainer } from '@trussworks/react-uswds';
+import PropTypes from 'prop-types';
 
 function Graphic({ graphics }) {
```

Comment:
To distinguish when new components are being made. You may have another function in the same file, but it can be named `const` to distinguish the actual component from a helper function.

Reply:
What resources suggest doing that?

Reply:
No resources were used; it's something I thought of myself.

Reply:
Okay - I think it would be better to consistently use es6 syntax